src/app/feed/loading.tsx

```diff
@@ -0,0 +1,21 @@
+export default async function Loading() {
+  return (
+    <>
+      <div className="h-16 w-full"></div>
```

Comment:
do we really need an empty div?

Reply:
ah yes for some reason the pt or the mt class is not working! I'll check again. to see it every things working without the div